### PR TITLE
Email

### PR DESCRIPTION
**Pull request by mikebrowntsbod**
@m-hogue please review

### CODE REVIEW

**Comment by m-hogue** [comment]
I don't miss PHP.

File: 1 [hunk 1]
```diff
@@ -0,0 +1,220 @@
+<?php
```
Comment (mikebrowntsbod) [inline 1]: I did a :wq 1 on accident and did not realize it

File: mail.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+							<td>" . $discussiontime . "</td>
+							<td>" . $discussiontext . "</td>
+						</tr>";
+			}//end of while
+			$body .= "</table>";
+
+			$body .= "</body>
+				</html>";
+		}//end of foreach
+
```
Comment (mikebrowntsbod) [inline mail.php]: These are now configured. They are pulled from the database they were hard-coded as a test for a proof of concept.

File: 1 (hunk 1 above)
Comment (m-hogue) [inline 1]: I had actually done this review yesterday before you merged it, but apparently submitting the review is an additional step. my b

File: mail.php (hunk 2 above)
Comment (m-hogue) [inline mail.php]: Cool. Just wanted to bring it to your attention.

File: 1 (hunk 1 above)
Comment (mikebrowntsbod) [inline 1]: It is okay. Learning process for me as well. I will look into the dao.

File: mail.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+			$body .= "<br /><br />
+					<h2>Current Votes</h2>
+					<table border=\"1\" width=\"100%\">
+					<tr>
+						<th>User</th>
+						<th>Date</th>
+						<th>Vote</th>
+					</tr>";
+
```
Comment (mikebrowntsbod) [inline mail.php]: Do you have any suggestions of what to review DAOs or just do a Google Search for PHP DAO configuration/setup?

File: 1 (hunk 1 above)
Comment (m-hogue) [inline 1]: Brief summary of what it is: https://en.wikipedia.org/wiki/Data_access_object

File: mail.php (hunk 3 above)
Comment (m-hogue) [inline mail.php]: Just do a google search. Here's something i found: http://www.sitecrafting.com/blog/php-patterns-part-ii/

File: 1 (hunk 1 above)
Comment (mikebrowntsbod) [inline 1]: What about this for example if you scroll down

http://www.sitecrafting.com/blog/php-patterns-part-ii/

**Pull request merged**